### ISSUE TITLE
Rows call on date field w/o standard view don't work

### ISSUE THREAD

**Issue by jaffee** [pilosa/pilosa#1783]
### Expected behavior

I guess `Rows` needs to take a date range and then give back all the rows which have set bits in the views that fall in that date range.

### Actual behavior

`Rows` doesn't have any notion of date and always queries the standard view. Now that we allow disabling the standard view, it doesn't work.

### Steps to reproduce the behavior

Import into a date field with the standard view disabled and then execute a Rows query.

### Success Criteria (implementation notes)

Basically we need to implement logic for merging the Rows results from multiple different views. Then for time quantum fields with no standard view, we'll merge together all rows from all the top-level quantum views. (e.g. if it's a YMD field, we just need to merge the results from each year... not the MD views)

**Comment by dmibor** [comment]
+1 to this - seems like this should be the main thing stopping from having GroupBy for our time fields without standard view

**Comment by travisturner** [comment]
@dmibor I submitted #1851 to address this. I haven't tested it wrapped by a `GroupBy` yet, but it's there if you want to take a look at it.
